ruby-challenges.rb

```diff
@@ -14,20 +26,39 @@
 
 # -------------------2) Create a method that takes in a string and removes all the vowels from the string. Use the test variables provided. HINT: Check out this resource: https://ruby-doc.org/core-2.6/String.html#method-i-delete
 
+def removes_vowels (string)
+    # string.delete 'a','e','i','o','u'
+    string.delete ('aeiouAEIOU')
+end
+
 album1 = 'Rubber Soul'
 # Expected output: 'Rbbr Sl'
 album2 = 'Sgt Pepper'
 # Expected output: 'Sgt Pppr'
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
-
+p removes_vowels album1
+p removes_vowels album2
+p removes_vowels album3
 
 # -------------------3) Create a method that takes in a string and checks if the string is a palindrome. A palindrome is the same word spelled forward or backward. Use the test variables provided.
 
+def palindrome_string string
+    if string.downcase == string.downcase.reverse
+        "#{string} is a palidrome"
+    else 
+        "#{string} is not palidrome"
+    end
+end
+
 palindrome_tester1 = 'Racecar'
 # Expected output: 'Racecar is a palindrome'
 palindrome_tester2 = 'LEARN'
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+p palindrome_string palindrome_tester1 
+p palindrome_string palindrome_tester2 
+p palindrome_string palindrome_tester3 
```

Comment:
Awesome job!  Looks like you and Ruby are friends!  Excellent work!

Reply:
Yes, Ruby makes more logical sense to me at this point in my development than JS.